### PR TITLE
Enforce Package name approval for preview release

### PR DESCRIPTION
**Pull request by praveenkuttappan**
APIView returns 202 is package is not approved for preview release. Package must be approved (basic approval like package name) in APIView to release a package as preview if it was never approved or released as GA before.

### CODE REVIEW

File: eng/scripts/Create-ApiReview.ps1 [hunk 1]
```diff
@@ -117,6 +117,13 @@ function ProcessPackage($PackageName)
                 }
                 elseif ($version.IsPrerelease)
                 {
+                    # Check if package name is approved. Preview version cannot be released without package name approval
+                    if ($respCode -eq '202' -and $pkgInfo.ReleaseStatus -and $pkgInfo.ReleaseStatus -ne "Unreleased")
+                    {
+                        Write-Host "Package $($PackageName) is not yet approved on APIView for preview release. Package approval for beta relase must be completed by an API approver if it was never released as a stable version."
```
Comment (benbp) [inline eng/scripts/Create-ApiReview.ps1:123]:
```suggestion
                        Write-Host "Package $PackageName is not yet approved on APIView for preview release. Package approval for beta relase must be completed by an API approver if it was never released as a stable version."
```

File: eng/scripts/Create-ApiReview.ps1 [hunk 2]
```diff
@@ -117,6 +117,13 @@ function ProcessPackage($PackageName)
...
+                        Write-Host "You can check http://aka.ms/azsdk/engsys/apireview/faq for more details on package name approval."
```
Comment (benbp) [inline eng/scripts/Create-ApiReview.ps1:124]: If write-error stops execution, then maybe `Write-Warning` or `throw <message>` instead?

```suggestion
                        Write-Error "You can check http://aka.ms/azsdk/engsys/apireview/faq for more details on package name approval."
```

**Review by benbp — APPROVED** [review]
a couple minor nits but LGTM

**Comment by praveenkuttappan** [comment]
@scbedd FYI. Python release for a preview version will start failing once this PR is merged if that package was never GAed before and require approval from architects. I will be sending out an email but it's a heads up in case if anyone reaches out about this error directly.

**Pull request merged**